flake.nix

```diff
@@ -2,36 +2,41 @@
   description = "A container and dev environment for medical imaging data science";
 
   # Nixpkgs / NixOS version to use.
-  inputs.nixpkgs.url = "nixpkgs/851558501d28bd3";
-  inputs.nixpkgs-old.url = "nixpkgs/4c3c80df545e";
-  inputs.nixGL.url = "github:guibou/nixGL";
-  inputs.nixGL.inputs.nixpkgs.follows = "nixpkgs";
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
```

Comment:
I'm on the fence about this change (using the branch name and not a specific commit hash), on one hand it is cleaner to refer to a branch so that the lock can be upgraded using the nix cli, but I'm a bit paranoid because it seems too easy to accidentally modify the lock. It's all under git so you can always revert the lock if you notice it changed underneath you though.

Reply:
I think once `nix build` creates a gcref then this style will be fine, but I'm open to changing it back if you want.

Reply:
I think we should probably do it this way, it's the less surprising way for sure. Even without gcroots we should always commit changes to the flake.lock so reverting is easy enough